building/Windows.md

```diff
@@ -0,0 +1,133 @@
+# Windows
+
+If we want we can build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows with Cygwin and WSL.
+
+## Cygwin
+
+In the beginning, we need to download libUSB and set the QEMU environment.
+
+To do that we need to simply download QEMU from the official website which is targeted to our system. The same
+situation is with libUSB, but we need to follow the instructions provided by the creator.
+
+* [QEMU](https://www.qemu.org/download/#windows)
+* [libUSB](https://github.com/libusb/libusb)
+
+Next up we need to download Cygwin from the official website. It is a collection of Open Source tools that provide
+functionality similar to a Linux distribution on Windows.
+
+* [Cygwin](https://cygwin.com/install.html)
+
+The installation process is in this case also a package manager. After selecting mirror we can easily select what
+packages we want, but in larger groups, this can be annoying that's why we only select two elements (They will be
+installed with dependencies) which are mandatory to use package manager in Cygwin terminal.
+
+Those are `Lynx` and `Wget`. Thanks to them, we will be able to install `apt-cyg` and work with a package similar way
+to Linux.
+
+Cygwin has a problem with translating multiple elements at once. Is suggested to use commands separately
+
+``` bash
+lynx -source rawgit.com/transcode-open/apt-cyg/master/apt-cyg > apt-cyg
+
+install apt-cyg /bin
+
+apt-cyg install libhidapi-devel libgccpp1 libtool libusb-devel libvirt-client libvirt-devel mingw64-i686-gcc-core mingw64-i686-gcc-g++ mingw64-x86_64-gcc-core mingw64-x86_64-gcc-g++ gcc-core gcc-g++ libclang-devel nano vim patch picocom python3 python3-pip python3-sphinx python3-devel qemu-integration symlinks tty-clock usbutils virt-viewer make autoconf automake cmake libc-client-devel git
+```
+
+After all of that, we need to run Cygwin installation because it will download all the needed dependencies without
+needing to do this manually.
+
+The next step is to prepare the genext2fs library and clone a repository. We need to make manually genext2fs library
+because Cygwin doesn't support it, and it's mandatory in case of creating images for the ia32 system.
+
+To make it simply paste this into the Cygwin terminal (Same as before we want to use commands line by line):
+
+``` bash
+git clone --recursive https://github.com/bestouff/genext2fs
+
+cd genext2fs
+
+./autogen.sh
+./configure
+make
+make install
+make check
+
+cd
+rm -rf genext2fs apt-cyg
+
+git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project/
+```
+
+Now we can simply follow instructions about building toolchains remembering to use commands to build inside
+phoenix-rtos-project folder.

```

Comment:
Is it then worth repeating here if it's already in building/README.md? Wouldn't a comment with a "Windows" label be sufficient, should such arise?